unittests/rpc/handlers/NFTInfoTest.cpp

```diff
@@ -288,7 +288,7 @@ TEST_F(RPCNFTInfoHandlerTest, NonExistNFT)
 // normal case when only provide nft_id
 TEST_F(RPCNFTInfoHandlerTest, DefaultParameters)
 {
-    constexpr static auto correntOutput = R"({
+    constexpr static auto currentOutput = R"({
```

Comment:
I'm not sure about this but i think it meant to say `correctOutput` 🤣 
Either way works 👍